app/views/templates/search/search_object.rb

```diff
@@ -175,100 +189,99 @@ def content
-              :sections  => [
+            provenance: data_section( {
+              title: 'site.object.meta-label.source',
+              sections: [
                 {
-                  :title  => 'site.object.meta-label.publisher',
-                  :fields => ['proxies.dcPublisher'],
-                  :url    => 'aggregations.edmIsShownAt'
+                  title: 'site.object.meta-label.publisher',
+                  fields: ['proxies.dcPublisher'],
+                  url: 'aggregations.edmIsShownAt'
                 },
                 {
-                  :title   => 'site.object.meta-label.provider',
-                  :fields  => ['aggregations.edmProvider']
+                  title: 'site.object.meta-label.provider',
+                  fields: ['aggregations.edmProvider']
                 },
                 {
-                  :title   => 'site.object.meta-label.data-provider',
-                  :fields  => ['aggregations.edmDataProvider']
+                  title: 'site.object.meta-label.data-provider',
+                  fields: ['aggregations.edmDataProvider']
                 },
                 {
-                  :title   => 'site.object.meta-label.providing-country',
-                  :fields  => ['europeanaAggregation.edmCountry']
+                  title: 'site.object.meta-label.providing-country',
+                  fields: ['europeanaAggregation.edmCountry']
                 },
                 {
-                  :title   => 'site.object.meta-label.identifier',
-                  :fields  => ['proxies.dcIdentifier']
+                  title: 'site.object.meta-label.identifier',
+                  fields: ['proxies.dcIdentifier']
                 },
                 {
-                  :title   => 'site.object.meta-label.provenance',
-                  :fields  => ['proxies.dctermsProvenance']
+                  title: 'site.object.meta-label.provenance',
+                  fields: ['proxies.dctermsProvenance']
                 },
                 {
-                  :title   => 'site.object.meta-label.source',
-                  :fields  => ['proxies.dcSource']
+                  title: 'site.object.meta-label.source',
+                  fields: ['proxies.dcSource']
                 },
                 {
-                  :fields      => ['timestamp_created'],
-                  :format_date => "%Y-%m-%d",
-                  :wrap        => {
+                  fields: ['timestamp_created'],
+                  format_date: "%Y-%m-%d",
+                  wrap: {
                     t_key: 'site.object.meta-label.timestamp_created',
                     param: :timestamp_created
                   }
                 },
                 {
-                  :fields      => ['timestamp_updated'],
-                  :format_date => "%Y-%m-%d",
-                  :wrap        => {
+                  fields: ['timestamp_updated'],
+                  format_date: "%Y-%m-%d",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.